README.md

```diff
@@ -1,2 +1,77 @@
 # fetch
```

Comment:
I like the name, but there are other projects that use it already, including another reasonably popular open source library: https://github.com/github/fetch. Not sure if we care, but just thought I'd mention it.